src/controllers/authControllers.js

```diff
@@ -53,12 +54,12 @@ module.exports.signup_post = async (req, res) => {
   try {
     const user = new User({email, name, password, phoneNumber}); 
     let saveUser = await user.save(); 
-    const token = createToken(saveUser._id);
-    //console.log(token); 
+    const token = createToken(saveUser._id.toString());
+    //signupMail(saveUser)
```

Comment:
Also send the host and protocol information so that the same verify link works for development and production